packages/framework/fluid-static/src/types.ts

```diff
@@ -83,61 +89,55 @@ export interface ContainerSchema {
 }
 
 /**
- * Events that trigger when the roster of members in the Fluid session change.
- * Only changes that would be reflected in the returned map of {@link IServiceAudience}'s
- * {@link IServiceAudience.getMembers} method will emit events.
- *
- * @remarks
- *
- * The following is the list of events emitted.
- *
- * ### "membersChanged"
- *
- * The "membersChanged" event is emitted when a member is either added or removed.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "memberAdded"
- *
- * The "memberAdded" event is emitted when a member joins the audience.
- *
- * #### Listener signature
- *
- * ```typescript
- * (clientId: string, member: M) => void;
- * ```
- * - `clientId` - A unique identifier for the client
- *
- * - `member` - The service-specific member object for the client
- *
- * ### "memberRemoved"
+ * Signature for {@link IMember} change events.
  *
- * The "memberRemoved" event is emitted when a member leaves the audience.
+ * @param clientId - A unique identifier for the client.
+ * @param member - The service-specific member object for the client.
  *
- * #### Listener signature
+ * @see See {@link IServiceAudienceEvents} for usage details.
+ */
+export type MemberChangeListener<M extends IMember> = (clientId: string, member: M) => void;
```

Comment:
Might prefer using past tense here since the inspectable data is updated prior to the event firing.
```suggestion
export type MemberChangedListener<M extends IMember> = (clientId: string, member: M) => void;
```